upstream-community-operators/akka-cluster-operator/akka-cluster-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,255 @@
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: Application Runtime
+    certified: 'false'
+    containerImage: 'lightbend-docker-registry.bintray.io/lightbend/akkacluster-operator:latest'
+    createdAt: 2019-05-30T00:00:00Z
+    description: Run Akka Cluster applications on Kubernetes.
+    repository: https://github.com/lightbend/akka-cluster-operator
+    support: 'Lightbend, Inc.'
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "app.lightbend.com/v1alpha1",
+          "kind": "AkkaCluster",
+          "metadata": {
+            "name": "akka-cluster-demo"
+          },
+          "spec": {
+            "replicas": 3,
+            "template": {
+              "spec": {
+                "containers": [
+                  {
+                    "name": "main",
+                    "image": "lightbend-docker-registry.bintray.io/lightbend/akka-cluster-demo:1.0.2",
+                    "readinessProbe": {
+                      "httpGet": {
+                        "path": "/ready",
+                        "port": "management"
+                      },
+                      "periodSeconds": 10,
+                      "failureThreshold": 10,
+                      "initialDelaySeconds": 20
+                    },
+                    "livenessProbe": {
+                      "httpGet": {
+                        "path": "/alive",
+                        "port": "management"
+                      },
+                      "periodSeconds": 10,
+                      "failureThreshold": 10,
+                      "initialDelaySeconds": 20
+                    },
+                    "ports": [
+                      {
+                        "name": "http",
+                        "containerPort": 8080
+                      },
+                      {
+                        "name": "remoting",
+                        "containerPort": 2552
+                      },
+                      {
+                        "name": "management",
+                        "containerPort": 8558
+                      }
+                    ]
+                  }
+                ]
+              }
+            }
+          }
+        }
+      ]
+  name: akka-cluster-operator.v0.0.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - kind: AkkaCluster
+        name: akkaclusters.app.lightbend.com
+        version: v1alpha1
+        description: An example Akka Cluster app that provides cluster visualization.
+        displayName: Akka Cluster
+        resources:
+          - kind: ServiceAccount
+            version: v1
+          - kind: Role
+            version: v1
+          - kind: RoleBinding
+            version: v1
+          - kind: Deployment
+            version: v1
+          - kind: ReplicaSet
+            version: v1
+          - kind: Pods
+            version: v1
+          - kind: ConfigMaps # operator-sdk uses this for leader election
```

Comment:
I know David put this here to score 100% but here's what the CSV doco says about the `resources` section:

>Resources: Your APIServices will own one or more types of Kubernetes objects. These are listed in the resources section to inform your end-users of the objects they might need to troubleshoot or how to connect to the application, such as the Service or Ingress rule that exposes a database.

> It’s recommended to only list out the objects that are important to a human, not an exhaustive list of everything you orchestrate. For example, ConfigMaps that store internal state that shouldn’t be modified by a user shouldn’t appear here.